charts/rancher-csp-adapter/0.1.0+up0.1.0-rc6/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Rancher CSP Adapter
+  catalog.cattle.io/hidden: "true"
+  catalog.cattle.io/kube-version: '>= 1.22.0'
+  catalog.cattle.io/namespace: cattle-csp-adapter-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux
```

Comment:
Does this chart support hybrid RKE2 clusters with both linux and windows nodes? If it does, this should be `linux,windows` and node-selectors/tolerations will need to be added to make sure resources are schedules on linux nodes only (We do this for nearly all of our other charts). Just verifying this was purposely set to linux only and not missed.

Reply:
No, it's local cluster only, which does not support windows. It could support RKE2 linux local cluster.